src/components/map/Map.tsx

```diff
@@ -133,23 +84,27 @@ export default function Map() {
       fillOpacity: 0.7
     });
   };
-
+  const seroPrevalence = Translate("Seroprevalence")
```

Comment:
why assign variables here instead of just supplying Translate("Seroprevalence") right into the JSX?